### DISCUSSION TITLE
What’s included in ChistaData Customer Support?

### DISCUSSION THREAD

**Discussion by emoido** [ChistaDATA/Cloud#15] [❓FAQ] [answered]
What’s included in ChistaData Customer Support?

**Comment by TheraniAA** [comment, marked as answer]
ChistaData Customer Support offers technical assistance, a knowledge base, community forums, a ticketing system, and a service level agreement to ensure customers receive the necessary support and resolve any issues encountered while using ChistaData Cloud services
